Modules/getpath.c

```diff
@@ -1297,15 +1297,15 @@ calculate_zip_path(PyCalculatePath *calculate)
     PyStatus res;
 
     /* Path: <PLATLIBDIR> / "python00.zip" */
-    wchar_t *path = joinpath2(calculate->platlibdir_macro, L"python000.zip");
+    wchar_t *path = joinpath2(calculate->platlibdir_macro, L"python" Py_STRINGIFY(PY_MAJOR_VERSION) Py_STRINGIFY(PY_MINOR_VERSION) L".zip");
```

Comment:
I wrote commit ce4179d0226768988965ac238ad15791cd377def in 2013 to fix compilation with IRIX C compiler: decode bytes string with a function call to prevent L"wide" "bytes" concatenation in the C compiler.

Reply:
But we now live in the C99 world where the grass is green and concatenation of prefixed-strings and non-preficed strings is defined:

```
In translation phase 6, the multibyte character sequences specified by any sequence of adjacent character and identically-prefixed string literal tokens are concatenated into a single multibyte character sequence. If any of the tokens has an encoding prefix, the resulting multibyte character sequence is treated as having the same prefix; otherwise, it is treated as a character string literal. Whether differently-prefixed wide string literal tokens can be concatenated and, if so, the treatment of the resulting multibyte character sequence are implementation-defined.
```